tsdb/series_file.go

```diff
@@ -65,7 +65,11 @@ func (f *SeriesFile) Open() error {
 		p := NewSeriesPartition(i, f.SeriesPartitionPath(i))
 		p.Logger = f.Logger.With(zap.Int("partition", p.ID()))
 		if err := p.Open(); err != nil {
-			f.Close()
+			f.Logger.Error("Unable to open time series data",
```

Comment:
Please change this line to `"Unable to open series file"`

Reply:
changed as suggested